spec/fixtures/sample.js

```diff
@@ -10,4 +10,20 @@ var quicksort = function () {
   };
 
   return sort(Array.apply(this, arguments));
-};
\ No newline at end of file
+};
+
+foo({
+    atIndentTwo: true
+  },
+  atIndentOne
+);
+atIndentZero = true;
+
+bar({
+    atIndentTwo: true
+  });
+atIndentZero = true;
+
+bar({
+    atIndentTwo: true });
+atIndentZero = true;
```

Comment:
Instead of modifying the `sample.js` fixture, could you just use a multi-line string in your test?

```coffee
beforeEach ->
  editor.setText """
    foo({
        atIndentTwo: true
      },
      atIndentOne
    );
    atIndentZero = true;
  """
```